app/src/main/java/com/android/unio/model/event/Event.kt

```diff
@@ -47,7 +47,8 @@ data class Event(
     val price: Double = 0.0,
     val date: Timestamp = Timestamp(Date()),
```

Comment:
Since you're already modifying the Event class, could you also maybe modify it to include a startDate and an EndDate instead of simply one date?